app/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get update && \
       git \
       wget \
       unzip \
-      default-jdk
+      default-jdk && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Is this a best practice?

Reply:
I think so in that I see it a lot when looking at other docker files.  I think it is just clean up to make the image smaller but it might have more significance.